shapeworks_cloud/core/tasks.py

```diff
@@ -38,7 +38,8 @@ def edit_swproj_section(filename, section_name, new_df):
     else:
         data[section_name] = new_contents
     data['data'] = [
-        {k: v.replace('../', '').replace('./', '') for k, v in row.items()} for row in data['data']
+        {k: v.replace('../', '').replace('./', '') if type(v) == str else v for k, v in row.items()}
```

Comment:
This is fine since it is a str, but rather than `type(v) == str`, `isinstance(v, str)` is usually preferred because it handles subclasses.

Reply:
CHnaged in [2081711](https://github.com/girder/shapeworks-cloud/pull/319/commits/208171199b2384c90ebe1567bb7ae955d547555c)